### ISSUE TITLE
Adobe Flash should be replaced

### ISSUE THREAD

**Issue by anubisg1** [CiscoDevNet/yang-explorer#112]
The entire UI is based on Flash. Flash is dead (https://theblog.adobe.com/adobe-flash-update/) and multiple browsers decided to make it very hard to use it or soon impossible.

There is the need to rebuild the tool using a better UI framework not based on Flash

**Comment by r-bhikhie** [comment]
Yes, someone PLEASE update this to something more fitting in this day and age.
Unless there is some reason why you would want to PREVENT your userbase from easily using the provided YANG models.

**Comment by eckelcu** [comment]
We hear you @anubisg1 and @r-bhikhie, and agree. We are working on a solution we can provide before the end of the year, and hopefully much sooner than that.

**Comment by egarnel** [comment]
+100 on this.  Flash is dead

**Comment by ITJamie** [comment]
@eckelcu Is there any update on this?

**Comment by eckelcu** [comment]
Nothing helpful to report at this time, @ITJamie , but still working on it.

**Comment by pistell** [comment]
It's been a few months, any update? I read that Flash is going to be completely removed from Chrome in January

**Comment by robertcsapo** [comment]
@pistell this **repo** is end of support.

> "As of 18th August 2020 this application is no longer maintained. Developers are free to use the code as-is, including forking, but no more updates will be forthcoming from the project maintainers and any new or outstanding issues will neither be tracked nor responded to."

https://github.com/CiscoDevNet/yang-explorer/blob/master/README.md

**Comment by r-bhikhie** [comment]
Wow. Amazing. Is there any alternative coming?

**Comment by ObjectIsAdvantag** [comment]
YANG Suite is the alternative, planning to ship in early 2021.
Please check this webinar for more information about YANG Suite: https://www.brighttalk.com/webcast/17628/455586
